clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnection.java

```diff
@@ -492,4 +533,26 @@ public void onClosed(Consumer<Exception> callback) {
 
         onClosedCallbackList.add(callback);
     }
+
+    private class ConnectionState implements InvocationBinder {
```

Comment:
How about `ConnectionBinder` instead of `ConnectionState` to avoid the name collision?

Reply:
Because it's going to have more later https://github.com/aspnet/SignalR/blob/adc5706bcb2b3cb85e98e303999eb9a778426f7c/src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs#L1106